connector/github/github.go

```diff
@@ -350,17 +350,17 @@ func (c *githubConnector) listGroups(ctx context.Context, client *http.Client, u
 		// 'teams' list in config.
 		if len(org.Teams) == 0 {
 			inOrgNoTeams = true
-			c.logger.Debugf("github: user %q in org %q", userName, org.Name)
+			c.logger.Infof("github: user %q in org %q", userName, org.Name)
 		} else if teams = filterTeams(teams, org.Teams); len(teams) == 0 {
-			c.logger.Debugf("github: user %q in org %q but no teams", userName, org.Name)
+			c.logger.Infof("github: user %q in org %q but no teams", userName, org.Name)
```

Comment:
@ericchiang we might want to keep this. it logs info about a situation downstream to [the user not in an org](https://github.com/estroz/dex/blob/c4c11bdf340da5edf7114c8c1681b5c1f7487ad6/connector/github/github.go#L545) that could be relevant to someone looking at logs.